src/shared/hooks/wallets.ts

```diff
@@ -169,15 +169,17 @@ export function useWalletOnboarding(): {
   const { value, updateStorage } =
     useLocalStorageChange<string>(LOCAL_STORAGE_WALLET)
 
+  const reflect = useReflect(false)
```

Comment:
Seems like reflect is used more like a context here? (not 100% sure about it tho)